.github/workflows/ci.yaml

```diff
@@ -22,51 +48,47 @@ jobs:
           node-version: 18
 
       - name: Install Clean dependencies
-        working-directory: ./angular
         run: npm ci
 
       - name: Run headless tests
         id: run-tests
-        run: npm test -- --watch=false --browsers=ChromeHeadless 
-        working-directory: ./angular
-        continue-on-error: true
+        if: ${{ always() }}
```

Comment:
You can use just `always()` everywhere. 

Btw, I don't think that you need to add `always` to tests, because basically it makes no sense to run tests if you got errors on `install clean dependencies` step for example. So Let's leave it without any condition.

Reply:
Done